Plugins/Sycl/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# This module needs at least CMake 3.13.
+cmake_minimum_required( VERSION 3.13 )
+
+find_package(SYCL REQUIRED)
```

Comment:
Could you move the `find_package` call to the main `CMakeLists.txt` in the repository root. Within Acts the `find_package` calls are all listed at a single point and not distributed over many files so it is easier to see what are the requirements.

Reply:
I hope I did an OK job here.